source/_posts/how-to-automate-deployments-of-ruby-application-updates-using-capistrano.md

```diff
@@ -0,0 +1,211 @@
+---
+title: How to automate deployments of Ruby Application updates using Capistrano
+date: 2020-10-31 01:12:32
+author: Ashish Jajoria
+categories:
+- ["Ruby on Rails"]
+tags: 
```

Comment:
Can you put traffic monitoring and road safety keyowrd in tag section right now it is showing in bottom

Reply:
Keyword should linked with website